api/requirements.txt

```diff
@@ -0,0 +1,5 @@
+fastapi[standard]
```

Comment:
Also, I am not too sure but when you dockerise, I had to install fastapi first and then fastapi[standard] so it might be something to keep an eye out for (but maybe it was just me installing ti wrong)

Reply:
Have double checked this with you - seems it was just the dotenv thing.